net_printer/benchmarks.md

```diff
@@ -0,0 +1,27 @@
+# Benchmarks
+
+This page describes the suite of benchmark designs that are used to assess
+routing performance. The full list of benchmark designs, along with links to
+the original sources and some basic statistics is provided in the following
+table:
+
+|Source Benchmark Suite|Benchmark Name|LUTs|FFs|DSPs|BRAMs|
```

Comment:
Can you provide the total number of resources of each type available on the xcvu3p to give an idea of utilization?

Reply:
see: df5b46d86fa197681e7b30bd8a39280d63672d81